lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -805,11 +818,15 @@ class LockLayer : public BaseCmd
         auto stage = getStage();
         if (!stage)
             return false;
-        if (_lockIt) {
+        if (_systemLock) {
```

Comment:
Note: For the time being, the undo state of a system lock goes into fully unlock. In a future design iterations a system-lock may get changed to never be undoable to prevent accidental unlocks.

Reply:
Ideally, it should return to the previous lock state (for example: locked -> system locked -> undo -> locked).